OneCommunityGlobal/HGNRest
# chore: deleted all related test changes leaving current functionality

# Description
This PR removes all commits related to testing, leaving the current functionality of the development branch for merging with the main branch. Currently, there is a deployment issue in beta (production) environment were the app is breaking whenever development branch is merged into it. This PR will try to make it possible to merge functionality from development into main. Once this is successful, testing related commits will be brought back one by one and it will be easier to detect which one of them is breaking the app (my assumption is new dependencies being brought to the codebase could be the source of the problem).

## Related PRS (if any):
N/A

## Main changes explained:
- Delete all related testing changes (folders, configurations, dependencies, etc)

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. List users and create a user

## Screenshots or videos of changes:

## Note:
